src/main/java/com/basho/riak/client/api/commands/kv/MultiCommand.java

```diff
@@ -54,9 +54,8 @@ protected MultiCommand(Builder builder)
 
         Submitter submitter = new Submitter(operations, maxInFlight, cluster, future);
 
-        Thread t = new Thread(submitter);
-        t.setDaemon(true);
-        t.start();
+        new Thread(submitter).start();
+
```

Comment:
If the threads that are created here aren't kept track of and `join()`-ed correctly, they could keep the entire JVM from shutting down cleanly.

I am assuming that is why `setDaemon(true)` was used previously - it's the lazy way of not having to deal with clean shutdown.

Is there a way that at the end of the `MultiCommand` these threads can be cleaned up? Does a `Submitter` know when all of its work is done and is able to exit and let the thread complete?